We now pass --with-cpu option to eglibc this ends up with configure errors if we do
not pass a cpu which eglibc has support for in sysdeps

| checking sysdep dirs... configure: error: The 603e subspecies of powerpc is not supported.
| + bbfatal 'oe_runconf failed'
| + echo 'ERROR: oe_runconf failed'

We fix this by adding the 603e sub directories with Implies to generic
powerpc to overcome this error

Upstream-Status: Inappropriate [OE config related]

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Index: libc/ports/sysdeps/powerpc/powerpc32/603e/fpu/Implies
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ libc/ports/sysdeps/powerpc/powerpc32/603e/fpu/Implies	2011-12-05 22:18:14.151326027 -0800
@@ -0,0 +1 @@
+powerpc/powerpc32/fpu
Index: libc/ports/sysdeps/unix/sysv/linux/powerpc/powerpc32/603e/fpu/Implies
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ libc/ports/sysdeps/unix/sysv/linux/powerpc/powerpc32/603e/fpu/Implies	2011-12-05 22:16:11.751326040 -0800
@@ -0,0 +1 @@
+powerpc/powerpc32/603e/fpu
